ci/integration-tests.sh

```diff
@@ -10,8 +10,9 @@ cargo install --force --debug --path .
 
 echo "Running integration test for crate ${INTEGRATION}"
 
-git clone --depth=1 "https://github.com/${INTEGRATION}.git" checkout
```

Comment:
Couldn't this also be done with `--single-branch`?

Reply:
Yes. But downloading `tar.gz` archives is about 1 second faster than using git.

Reply:
So 1 second in a ~8 minute build? I think just using `git` should be ok then?

Reply:
I would also rather keep it the current way if there is only a one-second speedup...